ui/v2.5/src/utils/screen.ts

```diff
@@ -0,0 +1,6 @@
+const isMobile = () =>
+  window.matchMedia("only screen and (max-width: 768px)").matches;
```

Comment:
Shouldn't `768px` be `767px`? IIRC both `min` and `max` are inclusive. So if CSS is using a mobile first approach and using `min-width: 768px` then at a screen of `768px` both the `min-width` and `max-width` will be hit.